doc/user/content/sql/drop-type.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "DROP TYPE"
+description: "`DROP TYPE` removes a user-defined data type."
+menu:
+  main:
+    parent: 'sql'
+---
+
+{{< version-added v0.7.1 />}}
+
+`DROP TYPE` removes a user-defined [custom data type](../create-type). It cannot be used on default data types.
+
+## Syntax
+
+{{< diagram "drop-type.svg" >}}
+
+Field | Use
+------|-----
+_data_type_name_ | The name of the type to remove.
```

Comment:
You say "remove" here but "drop" elsewhere; these verbs should be the same throughout the doc